keymaster.spec

```diff
@@ -50,23 +65,29 @@ install -p -m 0644 cmd/keymasterd/customization_data/templates/footer_extra.tmpl
 install -p -m 0644 cmd/keymasterd/customization_data/templates/login_extra.tmpl %{buildroot}/%{_datarootdir}/keymasterd/customization_data/templates/login_extra.tmpl
 install -d %{buildroot}/%{_datarootdir}/keymasterd/customization_data/web_resources
 install -p -m 0644 cmd/keymasterd/customization_data/web_resources/customization.css %{buildroot}/%{_datarootdir}/keymasterd/customization_data/web_resources/customization.css
-%pre
+
+%pre server
 /usr/bin/getent passwd keymaster || useradd -d /var/lib/keymaster -s /bin/false -U -r  keymaster
 
-%post
+%post server
 mkdir -p /etc/keymaster/
 mkdir -p /var/lib/keymaster
 chown keymaster /var/lib/keymaster
 systemctl daemon-reload
 
-%postun
+%postun server
 /usr/sbin/userdel keymaster
 systemctl daemon-reload
 
+
 %files
+
+%files client
+%{_bindir}/keymaster
+
+%files server
 #%doc
 %{_sbindir}/keymasterd
-%{_bindir}/keymaster
 %{_bindir}/keymaster-unlocker
 /usr/lib/systemd/system/keymaster.service
```

Comment:
Should these be under `%files client` as well? (since they are keymaster, not keymasterd)

Reply:
no, the service name is called keymaster. And is precisely what why are trying to split. Clientd do not need the systemd parts.